lib/archethic/mining.ex

```diff
@@ -34,7 +34,8 @@ defmodule Archethic.Mining do
   # version 5->6 the POI changed and is now done with tx.data.recipients.args serialized with :extended mode
   # version 6->7 add Add consumed inputs in tx.validation_stamp.ledger_operations
   # version 7->8 movement resolved address are now the genesis address of the destination
-  @protocol_version 8
+  # version 8->9 add proof of validation workflow and field tx.proof_of_validation
+  @protocol_version 9
```

Comment:
remainder: both this and https://github.com/archethic-foundation/archethic-node/pull/1588/files use v9 so they need to go in same release